SK.WebApp/src/services/consts.ts

```diff
@@ -0,0 +1,29 @@
+const apiAddress = "https://localhost:44324";
+
+enum anyLangageCodeKeys {
+    polish = 'POLISH',
+    english = 'ENGLISH',
+};
+
+type languageCodeMap = {
+    [key in anyLangageCodeKeys] : string;
+}
+
+const anyLanguageCode: languageCodeMap = {
+    POLISH: "pl-PL",
+    ENGLISH: "en-US",
+};
+
+enum anyEndpointKeys {
+    login = 'LOGIN',
+};
+
+type endpointMap = {
+    [key in anyEndpointKeys]: string;
+};
+
+const anyEndpoint: endpointMap = {
+    LOGIN: "/{languageCode}/api/User/login",
+};
+
+export { anyLanguageCode, anyLangageCodeKeys, anyEndpoint, anyEndpointKeys, apiAddress };
```

Comment:
Please export modules when you declare them, it's so much easier to reason about. Also could you add missing newline here? Did you install `ESLint` or `Prettier`? You can easily run those by using `yarn format:fix` and `yarn lint:fix` as described in `README.md` and `package.json`

Reply:
Ok, I change this. And yes I have install `ESlint `and `Prettier`, but in this project it's not working perfectly for me. When I do `yarn format:fix` and `yarn lint:fix` i have success but in logs I don't see all files.

I will try resolve this problem, before next push my changes.

Reply:
this command fix my code 'yarn prettier --write src/**/*.ts{,x}'

Reply:
better commands: `yarn prettier --write **/*.{ts,tsx,html}` and `yarn prettier --write **/*.{ts,tsx,html}`